docs/overhetalgoritmekader/index.md

```diff
@@ -1,83 +1,34 @@
-Dit kan worden beschouwd als de eerste versie van het Algoritmekader. Het implementatiekader verantwoorde inzet van algoritmen is in oktober 2023 hernoemd naar 'het Algoritmekader'. 
-
-In oktober 2023 is gestart met de doorontwikkeling van het Algoritmekader, zowel op inhoud als op vorm. Dat betekent dat inhoudelijke toevoegingen worden gedaan (denk aan hetgeen voortkomt uit de AI-verordening) en er wordt onderzocht hoe informatie zo optimaal mogelijk aan gebruikers getoond kan worden. 
-Het doel is dat het Algoritmekader gebruikers op een praktische wijze helpt bij het uitvoeren van hun werkzaamheden. 
-Het interactief kunnen doorzoeken van de informatie is daar een voorbeeld van.  
-
-Eind 2024 wordt de tweede versie van het Algoritmekader opgeleverd. 
-Dan moeten de vereisten en de maatregelen zijn uitgewerkt voor zover dan bekend. 
-Hierna zal het Algoritmekader in 'beheer' worden genomen. Dat betekent dat doorlopend de laatste ontwikkelingen, in afstemming met de omgeving, worden toegevoegd zodat het Algoritmekader actueel en betrouwbaar blijft.  
-
-## Samenwerking met de omgeving
-Een belangrijk uitgangspunt van het Algoritmekader is dat het kader op een open en transparante manier wordt ontwikkeld. 
-Hierbij wordt gebruik gemaakt van verschillende communicatiekanalen, zoals een communityplatform [Pleio](https://algoritmes.pleio.nl/), GitHub, periodieke bijeenkomsten en (waar passend) werkgroepen. 
-Informatie over algoritmes en AI worden uitgewerkt. 
-Zodra deze informatie op 'niveau' is, wordt dit in Github geplaatst en daarmee 'gepubliceerd'. 
-Dit noemen we *releases*. 
-
-De motivatie om op deze ‘open source’ manier aan dit Algoritmekader te werken is enerzijds de omgeving van deze informatie te voorzien en anderzijds om de omgeving uit te nodigen om hier feedback op te geven. 
-Werken in Github is voor het team Algoritmekader nieuw en experimenteel. 
-Het biedt de mogelijkheid voor alle geïnteresseerden om hun kennis en zienswijze te delen. 
-Daarmee kan (nieuwe) kennis snel onderdeel worden gemaakt van het Algoritmekader. 
-Daardoor blijft het Algoritmekader zo actueel mogelijk en wordt de inhoud breed gedragen.
-Tegelijkertijd vraagt dit om een aangepaste werkwijze en is hier bepaalde expertise voor nodig. 
-Het begin is gemaakt en het team Algoritmekader is nog lerende om hier optimaal invulling aan te geven. 
-Voor nu betekent dit concreet dat het langer kan duren voordat wordt gereageerd op suggesties of toevoegingen. 
-Er wordt gewerkt aan het inrichten van goede processen om snel en kundig te reageren. 
+# Over het algoritmekader 
+Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin zoek je de regels en adviezen op per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een testversie. De definitieve versie verschijnt eind 2024.
```

Comment:
Ook hier twijfel ik over het woord 'regels'. Klinkt toch een beetje alsof BZK regels oplegt, terwijl er geen nieuwe verplichtingen in het algoritmekader staan. Het is een overzicht van (de belangrijkste) bestaande vereisten volgens wet- en regelgeving of bestaand beleid, aangevuld met praktische suggesties over hoe je daar invulling aan kan geven.  Er is daar in de omgeving best wat onduidelijkheid over, dus ik vind het belangrijk dit ook duidelijk op de site te vermelden.

Reply:
```suggestion
Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin vind je de wetten en regels, hulpmiddelen en adviezen per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een bètaversie. De definitieve versie verschijnt eind 2024.
```